int64 numberToAverage
---
sensor_msgs/LaserScan scan